docs/windows_developer_guide.md

```diff
@@ -58,26 +42,17 @@ https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-fo
 Press Start and paste "Developer Command Prompt for VS 2022" to run cmd.exe for VS C++ developers
 Run commands in this prompt is not stated otherwise.
```

Comment:
It looks like all commands below could or should be executed in  Developer Command Prompt for VS 2022. Maybe this should be in bold here?